app/pages/order-task-list/commonManifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "backLinkText": "Go back",
+  "deleteOrderButton": {
+    "text": "Delete order",
+    "disabledAltText": "The delete order button will be disabled until you complete and save step 1"
+  },
+  "previewOrderButton": {
+    "text": "Preview order summary",
+    "disabledAltText": "The preview order button will be disabled until you complete and save step 1"
+  },
+  "submitOrderButton": {
+    "text": "Submit order",
+    "disabledAltText": "The submit order button will be disabled until the necessary steps are complete"
```

Comment:
why is it disabled alt text here but alt text in the context creator?

Reply:
I was kind of in turmoil here. So the manifest should contains the constants of what these values are. The altText we use will be dynamic based upon whether it's new or existing or whatever. So I am using this constant here to set the altText to this when required. Otherwise use something else.

Reply:
ah I see